Lib/shutil.py

```diff
@@ -283,6 +283,8 @@ def copyfile(src, dst, *, follow_symlinks=True):
                 return dst
 
             copyfileobj(fsrc, fdst)
+        fsrc.close()
+        fdst.close()
```

Comment:
open(), when used as a context manager, closes the files on exit. See https://docs.python.org/3/tutorial/inputoutput.html#tut-files